challenges/objects-arrays.js

```diff
@@ -14,17 +14,41 @@
 
 // Using your dinosaur objects, log answers to these questions:
 
+
+ const tyrannosaurus = {
+  objectName: 'tyrannosaurus',
+   diet: 'carnivorous',
+   weight: 7000,
+   length: '12k',
+   period: 'late cretaceious'
+} 
+
+ const stegosaurus = {
+ objectName: 'stegosaurus',
+ diet: 'herbivorous',
+   weight: '2000k',
+  length: 9,
+ period: 'late jurassic'
+ }
+
+ const velociraptor = {
+  objectName: 'velociraptor',
+  diet: 'carnivorous',
+ weight: '15k',
+  length: 1.8,
+ period: 'late cretaceious'
+};
 // How much did tyrannosaurus weigh?
-console.log();
+console.log(tyrannosaurus.weight);
 
 // What was the diet of a velociraptor?
-console.log();
+console.log(velociraptor.diet);
 
 // How long was a stegosaurus?
-console.log();
+console.log(stegosaurus.length);
 
 // What time period did tyrannosaurus live in?
-console.log();
+console.log(tyrannosaurus.period);
 
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
```

Comment:
Be sure to add the roar method. Refer to past projects if you're stuck, and if you still can't figure it out, reach out to me!